packages/api-docs/config/api-docs.js

```diff
@@ -13,6 +13,7 @@ module.exports = {
     path.resolve(__dirname, '../../date-time-utilities/dist/date-time-utilities.api.json'),
     path.resolve(__dirname, '../../merge-styles/dist/merge-styles.api.json'),
     path.resolve(__dirname, '../../react/dist/react.api.json'),
+    path.resolve(__dirname, '../../react-button/dist/react-button.api.json'),
```

Comment:
You'll probably also need to add an entry for `react-internal-compat.api.json` to get the compat button stuff to show up

Reply:
Talked with @ecraig12345 offline. We'll address in a separate PR. Made an issue tracking this in #15600.